doc/release-notes/9686-move-harvesting-client-id.md

```diff
@@ -0,0 +1 @@
+With this release the harvesting client id will be available for harvested files. A database update will copy the id to previously harvested files./
```

Comment:
Thinking about it, maybe we could add something like `This will make looking up harvested vs. local files in the database simpler: ... But make sure to update any existing database scripts you may have`.
I'll think about it/we can revisit this when putting together the notes for the release.